lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -121,7 +121,7 @@ describe('Fraggle Rock API', () => {
 
       const {auditResults, erroredAudits, failedAudits} = getAuditsBreakdown(lhr);
       // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`24`);
+      expect(auditResults.length).toMatchInlineSnapshot(`25`);
```

Comment:
Should this be higher? More than one audit depends on `LinkElements`.

Reply:
`URL` hasn't been converted from a base artifact yet, so those won't run. I *think* there's only 1 audit that uses *only* `LinkElements`, so that might check out.

Reply:
Ah yup. `preload-as` and `apple-touch-icon` require only `LinkElements`. `preload-as` is not in the default config though.